pkg/alertmanager/statefulset_test.go

```diff
@@ -1283,6 +1283,53 @@ func TestEnableFeatures(t *testing.T) {
 	}
 }
 
+func TestValidateAdditionalArgs(t *testing.T) {
```

Comment:
We can have a single test ensuring that additional args are passed down to the statefulest but no need to check on the version (see the other comment above).